### PR TITLE
Videostream

### PR DESCRIPTION
**Pull request by elbrujo1987**
👍

### CODE REVIEW

File: bundle/utils.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        return function() {
+            if (!warned) {
+                warned = true;
+                console.warn(msg);
+            }
+            return cb.apply(this, arguments);
+        }
+    },
+    isU8: function(obj) {
+        return obj instanceof Uint8Array || Buffer.isBuffer(obj);
```
Comment (jimmywarting) [inline bundle/utils.js:60]: A buffer is a instance of Uint8Array so `|| Buffer.isBuffer(obj)` is unnecessary

**Comment by diegocr** [comment]
What's going on here? :-)

Be aware our fork cannot be merged here as-is, it will break this badly and any npm dependents.

Also, let me point out we don't endorse nor know who is this @elbrujo1987

**Comment by jhiesey** [comment]
Sorry, I haven't been putting much time into maintaining this module recently. I'll look at it this weekend.

**Comment by diegocr** [comment]
No worries @jhiesey, we know what that does mean ;-)

What i would personally prefer is this PR closed, given the OP is not involved in any way with the upstream `meganz:master`

**Comment by jhiesey** [comment]
@elbrujo1987 I don't think this is appropriate to merge in as it is without more significant discussion.

I'm not opposed to making a new major version based on https://github.com/meganz/videostream I just took a quick look at the code and it looks like it has lots of new good stuff, but no documentation about what was added.

@diegocr it looks like you did most of the work on that; if you're interested in documenting your changes I could add you as a maintainer on my repo. Let me know.

**Comment by diegocr** [comment]
Thanks @jhiesey i will consider it, our company fork is heavily tied to our underlying [web-client codebase](https://github.com/meganz/webclient), so back-porting it to how yours does work won't be quite straight forward i guess, we may could add different layers to work in a way or another, although that will take time to do as well.

**Pull request closed without merging**